watertap/property_models/unit_specific/cryst_prop_pack.py

```diff
@@ -1740,9 +1743,9 @@ def rule_dh_crystallization_mass_comp(b):
     def _enth_mass_solute(self):
         self.enth_mass_solute = Var(
             ["Sol"],
-            initialize=1e3,
-            bounds=(1e-3, 1e4),
-            units=pyunits.kJ / pyunits.kg,
+            initialize=1e6,
+            bounds=(1e-6, 1e7),
```

Comment:
It looks like you decreased the lower bound by 3 orders of magnitude instead of increasing it, unlike all of your previous changes which increase bounds by 3 orders of magnitude.

Reply:
That was an oversight and has been fixed. Thanks for pointing it out.